scimono-server/src/main/java/com/sap/scimono/entity/validation/patch/PatchValidationFramework.java

```diff
@@ -61,7 +61,7 @@ public void validate(PatchBody body) {
     		  .getAttributes()
     		  .stream()
     		  .map(Attribute::getName)
-    		  .filter(attributeName -> attributeName.equalsIgnoreCase(operation.getPath()))
+    		  .filter(attributeName -> (attributeName != null && attributeName.equalsIgnoreCase(operation.getPath())))
```

Comment:
I don't think that this should be the fix. Maybe you should expect that `schemaAPI` returns you the correct data and fails if not. Otherwise, using this way you hide possible backend error